hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1078,6 +1189,19 @@ components:
       type: string
       example: "1586567700.453054000"
       pattern: '/^\d{1,10}(\.\d{1,9})?$'
+    TimestampRange:
+      type: object
+      description: A timestamp range an entity is valid for
+      properties:
+        from:
+          type: string
```

timestamp range from and to should use timestamp type:
```suggestion
          $ref: '#/components/schemas/Timestamp'
```